cmd/edge-proxy/main.go

```diff
@@ -59,6 +60,7 @@ func main() {
 	flag.StringVar(&certStrategy, "cert-strategy", fog_tls.DefaultDriver(), fmt.Sprintf("Certificate strategy must be one of: %v", fog_tls.Drivers()))
 	flag.Var(&certStrategyOptions, "cert-strategy-options", "Can be specified one or more times. Must be a key-value pair (<key>=<value>)")
 	flag.StringVar(&forwardingAddressesMap, "forwarding-addresses", "{}", "Map of local address to forwarded address for outgoing HTTP requests. For each forwarding request received at proxy-listen, the destination URI in the request is rewritten based on this map, where the destination server is replaced with the value of the corresponding key.  If the destination server isn't found in this map, then the value of proxy-uri is used.  Must be a json string")
+	flag.StringVar(&httpsProxyAddr, "https-proxy-listen", "localhost:8888", "Listen address for HTTPS (CONNECT) proxy server")
```

Comment:
This is misleading.  This isn't an https listener.

Reply:
Would it be better if we called an "http-connect-proxy"?

Reply:
Renamed to HTTP Tunnel.